### PR TITLE
docs: updated switch.mdx

### PR DESCRIPTION
**Pull request by Ildest**
Please review and commit to main

### CODE REVIEW

**Comment by Ildest** [comment]
Hope this is OK now. I did som changes after review from Marianne, but I am not sure if those changes became a part of this file? I just continued to edit the same file, but not sure if that is correct?

File: packages/react/src/components/form/Switch/Switch.mdx [hunk 1]
```diff
@@ -27,38 +29,55 @@ Bruker [Fieldset](/docs/komponenter-fieldset--docs) til gruppering.
...
-du har behov for det.
+Bruk `position="right"` til å plassere `Switch` på høyre side av ledeteksten hvis
+du trenger det.
 
 <Canvas of={SwitchStories.FullWidthRight} />
 
-## Retningslinjer
+## Retningslinjer for `Switch`
+
+`Switch` skal være et valg mellom to alternativer. Vi bruker `Switch` til innstilinger, ikke i skjema.
```
Comment (Barsnes) [inline packages/react/src/components/form/Switch/Switch.mdx:39]:
```suggestion
`Switch` skal være et valg mellom to alternativer. Vi bruker `Switch` til innstillinger, ikke i skjema.
```

File: packages/react/src/components/form/Switch/Switch.mdx [hunk 2]
```diff
@@ -27,38 +29,55 @@ Bruker [Fieldset](/docs/komponenter-fieldset--docs) til gruppering.
...
-Vi bruker switch når brukeren skal kunne slå noe av eller på. Switch er den digitale varianten av av/på-brytere, for eksempel en lysbryter. Et eksempel fra telefon er Flymodus av/på.
+- aktivere eller deaktivere funksjoner/tilstander med en gang
+- slå varsler av
+  og på
+- endre visning av innhold mellom to kategorier, for eksempel se
+  statistikk per måned eller år
 
-Switch skal være et valg mellom to alternativer som utelukker hverandre, og komponenten skal alltid ha et standardvalg.
+  <br />
+  Passer ikke til
```
Comment (Barsnes) [inline packages/react/src/components/form/Switch/Switch.mdx:51]:
```suggestion
Passer ikke til
```

**Comment by mrosvik** [comment]
Thanks @Barsnes 🙏🏻

**Review by Barsnes — APPROVED** [review]

**Pull request merged**